plugin-packs/postcss-preset-env/src/index.js

```diff
@@ -113,14 +115,24 @@ const plugin = opts => {
 	const usedPlugins = supportedFeatures.map(feature => feature.plugin);
 	usedPlugins.push(stagedAutoprefixer);
 
+	const internalPlugin = () => {
```

Comment:
This was broken.
It's not possible to have both `OnceExit` and `plugins` in the same PostCSS plugin.

These need to be chained.

I do not know the `exportsTo` plugin so I can't easily add a test for it to double check that everything is fine now.